### PR TITLE
Subscription webhook

### PR DESCRIPTION
**Pull request by Nnanyielugo**

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -299,6 +299,41 @@ def update_usernames():
     'failed': failed,
   }
 
+@app.route("/spec")
+def get_apispec():
+    return jsonify(spec.to_dict())
+
+@app.route('/account/credit_balance', methods=['GET'])
```
Comment (mjmayank) [inline app/__init__.py:306]: can you delete this actually while we're doing this, it was for something else

**Review by mjmayank — APPROVED** [review]
apart from deleting the herc thing

File: app/__init__.py (hunk 1 above)
Comment (Nnanyielugo) [reply app/__init__.py:306]: @mjmayank do you mean the `get_apispec` endpoint?

Comment (mjmayank) [reply app/__init__.py]: no the /account/credit_balance

**Pull request merged**